python-regression/README.md

```diff
@@ -1,33 +1,140 @@
+that contains more milestones than the `local-snapshots-pruning-depth`. This node is checked to make sure that after 
+starting, transactions that should be pruned form the database have been pruned correctly.   
+
+### Running Tests on TIAB (Requires present kubernetes configuration)
+These tests have been written to interact with a set of nodes with provided configurations, running on a `kubernetes` 
+cluster. The `ciglue.sh` script will run all the available machines in the `./tests/features/` directory. The nodes will
+be configured based off of the `./tests/features/machine[x]/config.yml` file, and will update the 
+`./tests/features/machine[x]/output.yml` file with the newly created cluster host and port information. This script then
+proceeds to run each of the feature files in the respective machine directories, and these tests are run on the nodes 
+specified in the `output.yml` file. Once the tests have completed, the cluster is torn down. The default logging 
+of these tests is verbose, for debugging purposes. To use this automated script, simply use the following command: 
+```
+bash ciglue.sh [insert iri docker image here]
+```
+
+To run on the latest IOTA build, run:
+```
+bash ciglue.sh iotacafe/iri-dev:latest
+```
+
+### Running Tests Locally
+
+To run the tests using a local/custom node, the first step will be to define the nodes that you would like to run the 
+tests on. Each `aloe` test will run off of a given feature file, stored in the `./tests/features/machine[x]` directory.
+The tests rely on an `output.yml` configuration file, which contains the host and port information of the nodes that 
+the test will be run on. This file is generated automatically by the `TIAB` approach, but needs to be generated/filled 
+out manually for local tests. The `output.yml` files are found in the machine directory for the test, and should have a 
+structure as follows: 
+
 ```
-aloe
+nodes:
+  nodeA:
+    host: localhost
+    podip: localhost
+    ports:
+      api: 14265
+      gossip-udp: 14600
+      gossip-tcp: 15600
+      zmq-feed: 5556
+    clusterip_ports:
+      api: 14265
+      gossip-udp: 14600
+      gossip-tcp: 15600
+      zmq-feed: 5556
+
+  nodeB:
+    host: localhost
+    podip: localhost
+    ports:
+      api: 15265
+      gossip-udp: 14605
+      gossip-tcp: 15605
+      zmq-feed: 6556
+    clusterip_ports:
+      api: 15265
+      gossip-udp: 14605
+      gossip-tcp: 15605
+      zmq-feed: 6556
+
+``` 
+
+Once the `output.yml` configuration is set, the tests can be run.
+
+From the `python-regression` directory, a test can be run using the following command structure: 
 ```
-Extra Features
-If you would like a more verbose readout of the tests being run, you can add the -v flag:
+aloe [Feature file name]  -w [Location of feature file]  -v [Optional: verbose test name logging] --nologcapture 
+[Optional: release logging for individual tests for increased verbosity]
```

Comment:
The `[....]` looks like something we need to fill in when typing the command in the cli, or Optional according to linux `man`.
Maybe change it to something like:

aloe *Feature File Name*  -w *Location of feature file*  [-v] [--nologcapture]

-v - verbose test name logging
--nologcapture - release logging for individual tests for increased verbosity


In linux `man` there would be underlining and no italics. I just wasn't able to underline in the comments.